apps/condo/next.config.js

```diff
@@ -21,6 +22,7 @@ const withTM = withTMModule([
     '@emotion/styled',
 ])
 
+// TODO(antonal): DOMA-6804 implement validation of configs in bin script
```

Comment:
You should create a typed `.env` Proxy, please create ADR before, since I have an experience with `.env` validation, so we can find most `easy-to-use` case for our project

Reply:
ADR is already under work since Feb 2023
https://confluence.in.doma.ai/pages/viewpage.action?pageId=1214680731

Let's discuss it

Reply:
Extracted changes, related to ADR, into different branch, that will be pushed in another PR

Reply:
Extracted changes, related to ADR#16 into different PR, that will be opened later with task DOMA-7045